pkg/windows/connectivity.go

```diff
@@ -161,3 +168,8 @@ func (c *sshConnectivity) transfer(reader io.Reader, filename, remoteDir string)
 	}
 	return nil
 }
+
+func (c *sshConnectivity) isReachable() bool {
+	sshClient, err := c.newConnection()
```

Comment:
Why do you need to use a new client to check connectivity? Can't you use the existing client and try reaching the VM by executing a dummy command?

Reply:
true, a new client isn't needed. I'll simplify